styles/globals.css

```diff
@@ -34,9 +34,27 @@ a {
 }
 
 @layer components {
-  .styledDocument a{
+  .styledDocument a {
     @apply font-semibold text-darkBlue;
   }
+  .styledDocument h1 {
+    @apply text-[2em] font-semibold py-6 leading-10
```

Comment:
Why em and not tailwind here?

Reply:
@bdougie The tailwind class doesn't cover all the values I have used here. These em values are exactly the same value that h1, h2, etc inherit by default from HTML. So I considered em here in order to maintain consistency.

Reply:
TBH, we don't need anything past h3. This is setup for scalability but doesn't take into consideration that this page has not been designed by @Saucy-Christian.

I'd prefer if we used the defaults from tailwind than begging building our own custom framework.

Reply:
@bdougie Replaced with tailwind classes now.